src/videojs-hls.js

```diff
@@ -3,688 +3,518 @@
  * The main file for the HLS project.
  * License: https://github.com/videojs/videojs-contrib-hls/blob/master/LICENSE
  */
+ /*jshint -W092 */
+ /*jshint -W062 */
```

Comment:
Yes, please remove the overrides. The hint was telling you not to use tabs for indentation :)